src/main.rs

```diff
@@ -20,23 +21,29 @@ mod git;
 
 use cmd::call;
 use config::Config;
-use error::FatalError;
-use termcolor::Color::{Green, Blue};
-
-fn build(docs_path: &str, shell: &mut Shell, dry_run: bool) -> Result<(), FatalError> {
-    try!(shell.verbose(|s| {
-        s.status_with_color("", "Building Sphinx docs.", Blue)
-    }));
+use failure::{Error, SyncFailure};
+use termcolor::Color::{Blue, Green};
+
+fn build(docs_path: &str, shell: &mut Shell, dry_run: bool) -> Result<(), Error> {
+    // TODO: Remove the SyncFailure mapping once Cargo got updated. This is likely
```

Comment:
Could you make an GitHub issue for this instead? Just to avoid TODO comments.

Reply:
Sounds good!